Dockerfile

```
@@ -0,0 +1,15 @@
+FROM golang:1.22
+
+WORKDIR /app
+
+COPY go.mod go.sum ./
+
+RUN go mod download
+
+COPY *.go ./
```

Можно объединить в одну команду
```suggestion
COPY *.go *.db ./
```

---

> Можно в кабине одна

Спасибо!